### DISCUSSION TITLE
wrong documentation? cell editor doesn't work as described in the WIKI - it is not available and missing in context menu.

### DISCUSSION THREAD

**Discussion by aisbergde** [dbeaver/dbeaver#11459] [Q&A] [answered]
#### System information: 
- Operating system (distribution) and version: Windows 10
- DBeaver version 7.3.5.202102141704
- Additional extensions
- 
#### Connection specification:
- Database name and version: SQL Server 2019
- Driver name: I don't know where to find this information. dbeaver installed something when I first time connected to an SQL Server
- Do you use tunnels or proxies (SSH, SOCKS, etc)? no

#### Describe the problem you're observing:

The main reason why I wanted to try dbeaver was the feature "cell editor" which is described here:
https://github.com/dbeaver/dbeaver/wiki/Data-Viewing-and-Editing
I need this feature to enter multiline data into some text fields, but SSMS or Azure Data Studio have issues with multiline data input. And I prefer an open source project, so I could recommend to other users to use a working tool, when it will work for me.

**The cell editor is not available as described in the documentation.** There is no "edit cell" in the context menu and there is no "edit cell" button.
![image](https://user-images.githubusercontent.com/9020187/108404269-f121aa00-721f-11eb-9146-9ccc6b2f90a2.png)

#### Steps to reproduce, if exist:

In the tree, double-click on any table. The data grid will open.
![image](https://user-images.githubusercontent.com/9020187/108404543-49f14280-7220-11eb-8012-438bb4d70ddd.png)

I can't imagine that this feature was implemented is then removed. So I assume the feature is still there, but that it has been well hidden for new users.

Maybe the implementation has changed, but the documentation is wrong and describes an old way?
Even if it is possible - then it is not intuitive.

**Comment by uslss** [comment, marked as answer]
Hi @aisbergde ,
you can open the editor using Shift+Enter when the needed cell is chosen.
Also, you can change double-click behavior in preferences.
![image](https://user-images.githubusercontent.com/31996417/108873711-76380500-760c-11eb-927e-5be1d7eb72d5.png)

**Comment by aisbergde** [comment]
@uslss 

I don't understand why this issue was closed. May because I tagged it with the wrong "Question and Answer"? 
I spent time to write issues to improve the product, and the documentation is part of the product. And I pointed out that the documentation is wrong and should be corrected.

>Right-click the cell and click Inline edit on the context menu.

This is wrong. There is no "inline edit" on the context menu. And I think it would be a good idea if the documentation describes the actual state of the product. But instead of keeping the issue open or to correct the documentation the issue was just closed.

The product is great, the documentation is great. And I know how hard it is to keep the documentation up to date. And there will always be small issues. I as a developer am happy when I get feedback like this, that's why I give it. But if my feedback doesn't interest the developers, then I don't give it. So is there an interest here to work on improving the product, including the documentation, or are issues closed here just so the list gets shorter?
